backstage/redis.resources.yaml

```diff
@@ -0,0 +1,21 @@
+---
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: fxa-profile-cache
+  description: Stores aggregated profile data.
+
```

Comment:
nit: extra line break

Reply:
Yea, it's a common YAML formatting thing (prettier seems to do it).